### ISSUE TITLE
Add option to display layer name as title of list

### ISSUE THREAD

**Issue by allisonmuise** [Esri/crowdsource-polling#95]
In the list title area:
- add the layer name to the list header (left-justified)
- Layer name should display only when configured to appear.
- Add Boolean option to the config panel to show/hide layer title. Option should be 'off' by default to match current app behavior.

**Comment by MikeTschudi** [comment]
Where would this be displayed? The list header currently has a label and switch which can take up most of the space in English, let alone other languages:
![image](https://cloud.githubusercontent.com/assets/2125181/10060883/3b3242bc-6209-11e5-8d11-3f0d7160516b.png)

We could insert this layer name above the bar with the switch, e.g., in the yellow-highlighted area shown below; this space would only exist when the layer name is configured to appear.
![image](https://cloud.githubusercontent.com/assets/2125181/10060939/a4404786-6209-11e5-900b-b7872950d9a8.png)

**Comment by allisonmuise** [comment]
cc @ScottOppmann @NikkiGolding  @LindsayThomas 
Thoughts?

**Comment by MikeMillerGIS** [comment]
I would vote to not include the layer title and wait for a request to add it

**Comment by LindsayThomas** [comment]
@allisonmuise I agree with Mike. Because the application only looks at one layer there isn't a need to display the layer name.

**Comment by allisonmuise** [comment]
@ScottOppmann & @NikkiGolding : This was functionality you requested in order to provide a title for the list in desktop view. Do you still want to pursue this? If so, do you have a preference for one of Mike's suggestions?

**Comment by NikkiGolding** [comment]
@allisonmuise If it is configurable like @MikeTschudi  suggests it gives people the flexibility to add it if they choose. Above the bar works for me

**Comment by allisonmuise** [comment]
final decision was above the bar, configurable, and off by default.
